File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.go
func topKFrequent(nums []int, k int) []int {
	result := make([]int,0,k)
	hashMap := make(map[int]int)
	for _, val := range nums {
		hashMap[val]++
	}

	for i:=0; i<k; i++ {
		maxThing := 0
		elementMax := 0
		for element, value := range hashMap {
			if value > maxThing {
				maxThing = value
				elementMax = element
			}
		}
		result = append(result,elementMax)
		hashMap[elementMax] = 0
	}
	return result
}

File: Data Structures & Algorithms/isomorphic-strings/submission-2.py
class Solution:
    def isIsomorphic(self, s: str, t: str) -> bool:
        hashmap = {}

        for i in range(len(s)):
            if s[i] in hashmap and hashmap[s[i]] != t[i]:
                return False
            else:
                if s[i] not in hashmap and t[i] in hashmap.values():
                    return False

            hashmap[s[i]] = t[i]

        return True


        #b = b a = a d = b